docs/docs/en/getting-started/index.md

```diff
@@ -393,11 +393,11 @@ Once everything is set up, you can run your FastAgency application using the fol
             ```
 
         !!! danger "Currently not working on **MacOS**"
-            The above command is currently not working on **MacOS**, please use the alternative way of starting the application from below ([#362](https://github.com/airtai/fastagency/issues/362)).
+            The above command is currently not working on **MacOS**, please use the alternative way of starting the application from below ([#362](https://github.com/airtai/fastagency/issues/362){target="_blank"}).
 
     2. Using [Gunicorn](https://gunicorn.org/){target="_blank"} WSGI HTTP server:
 
-        The preferred way to run the Mesop application is using a Python WSGI HTTP server like [Gunicorn](https://gunicorn.org/){target="_blank"}. First, you need to install it using package manager such as `pip` and then run it as follows:
+        The preferred way to run the [**Mesop**](https://google.github.io/mesop/){target="_blank"} application is using a Python WSGI HTTP server like [**Gunicorn**](https://gunicorn.org/){target="_blank"}. First, you need to install it using package manager such as `pip` and then run it as follows:
```

Comment:
gunicorn or waitress, please see bellow for the problem with Windows

Reply:
We have a note below this paragraph explaining the problem in windows and the solution. We have added this common note in all required places.

Reply:
I know, but why don't we say 

The preferred way to run the [**Mesop**](https://google.github.io/mesop/){target="_blank"} application is using a Python WSGI HTTP server like [**Gunicorn**](https://gunicorn.org/){target="_blank"} or [**waitress**](something){target="_blank"}.